admin/src/js/controllers/authorization-roles.js

```diff
@@ -80,12 +80,14 @@ angular.module('controllers').controller('AuthorizationRolesCtrl',
       Object.keys($scope.roles).forEach(function(key) {
         changes[key] = $scope.roles[key];
       });
+
       changes[$scope.newRole.key] = {
         name: $scope.newRole.name,
         offline: $scope.newRole.offline
       };
       save(changes).then(function() {
         $scope.submitting = false;
+        $translate.refresh();
```

Comment:
This is the right function, but I think it's in the wrong place. The issue doesn't talk about roles at all so I don't think this controller should be touched.

Instead, consider refreshing when the language is updated. This happens in a few different places (eg: EditLanguageCtrl, EditTranslationCtrl, as well as medic-conf). You could either find everywhere the translation can be updated in the UI and add this refresh call there, _or_ add a db changes listener so any time a translation file is updated we refresh.

For now I think I'd just add `refresh` calls everywhere we change the language in the UI.